docs/src/main/sphinx/connector/hive.rst

```diff
@@ -99,6 +99,68 @@ Materialized views
 The Hive connector supports reading from Hive materialized views.
 In Trino, these views are presented as regular, read-only tables.
 
+Hive views
+----------
+
+Hive views are defined in HiveQL and stored in the Hive Metastore Service. They
+are analyzed to allow read access to the data.
+
+The Hive connector includes support for reading Hive views with three different
+modes.
+
+* disabled
```

Comment:
I'd remove these bullets and may be just rely on the sections below. 

Also, the 3 values "disabled", "legacy" and "experimental" don't have significance in terms of exact config values. So it might be better to write them as normal title words using `^^^^` and saying explicitly "Legacy Mode" and "Experimental mode".  In "disabled" case, we're not really supporting hive views, so may be just mention that separately instead of calling it a mode.

Reply:
They are three different distinct behaviors and need to be documented as such. The list makes this clear. I think it should stay as is. If anything we could make the three sections as subtitles but I did not want to make them that prominent.